src/Umbraco.Core/Services/Implement/ContentTypeServiceBaseOfTItemTService.cs

```diff
@@ -114,13 +114,13 @@ protected bool OnSavingContainerCancelled(IScope scope, SaveEventArgs<EntityCont
 
         protected void OnSavedContainer(IScope scope, SaveEventArgs<EntityContainer> args)
```

Comment:
I see you added the `CanCancel = false` to the args for OnRenamedContainer but not OnSavedContainer. I'll update.